Sources/PenguinStructures/AnyArrayBuffer.swift

```diff
@@ -28,6 +28,51 @@ public struct AnyArrayBuffer<Storage: AnyArrayStorage> {
 
   /// The type of element stored here.
   public var elementType: Any.Type { storage.elementType }
+
+  /// Returns the result of calling `body` on the elements of `self`.
+  public func withUnsafeRawBaseAddress<R>(
+    body: (UnsafeRawPointer)->R
+  ) -> R {
+    storage.withUnsafeMutableRawBufferPointer { b in
+      body(b.baseAddress.map { .init($0) } ?? UnsafeRawPointer(bitPattern: -1).unsafelyUnwrapped)
+    }
+  }
+
+  /// Returns the result of calling `body` on the elements of `self`.
+  public mutating func withUnsafeMutableRawBaseAddress<R>(
```

Comment:
```suggestion
  public mutating func withUnsafeMutableRawPointerToElements<R>(
```

Precedented in `ManagedBuffer` API.